tests/test_parunits.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+from astropy import units as u
+from pint.models.timing_model import UnknownParameter
+from pint.utils import get_unit
+
+
+@pytest.mark.parametrize(
+    "p",
+    [
+        "F0",
+        "F1",
+        "DM",
+        "DMX_0001",
+        "DMX_0002",
+        "DMXR1_0001",
+        "POSEPOCH",
+        "PMRA",
+        "PMELONG",
+        "FB0",
+        "FB12",
+        "PB",
+        "RA",
+        "A1",
+        "PLANET_SHAPIRO",
+        "M2",
+        "EDOT",
+        "ECC",
+        "OM",
+        "T0",
+        "TASC",
+        "XDOT",
+        "EFAC",
+        "EQUAD",
+        "JUMP1",
+    ],
+)
+def test_par_units(p):
+    unit = get_unit(p)
+    print(f"{p}: {unit}")
+    assert isinstance(unit, u.UnitBase) or unit is None
```

Comment:
I think this test can be split into two tests -- one for the case where the output is a unit, and one for the case where it is None.